src/main/java/com/epam/wl/services/TestLoginService.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.wl.services;
+
+import com.epam.wl.DBHelper;
+import com.epam.wl.dao.UserDAO;
+import com.epam.wl.entities.User;
+import com.epam.wl.enums.UserRole;
+
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class TestLoginService {
+    public static UserDAO userDAO = new UserDAO(DBHelper.getEmbeddedDatabase());
+
+    public static String signIn(String email, String password) throws SQLException {
+        if (userDAO.isLibrarian(email, password)) {
+            return "librarian_from_login.jsp";
+        }
+
+        return "user_from_login.jsp";
+    }
+
+    public static String addNewUser(String name, String lastname, String email, String passwordHash, UserRole userRole) {
```

Comment:
Why did you make all methods of this service static? It would be better to make it not static and inject instant of this service everywhere, where you use its methods. It'll be easier to track usage of this service